helmfile/charts/prometheus-alerts/templates/alerts/packets-dropped.yaml

```diff
@@ -0,0 +1,68 @@
+{{- if and .Values.defaultRules.create .Values.defaultRules.rules.networkpolicies }}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: {{ printf "%s-%s" (include "prometheus-alerts.fullname" .) "packets-dropped" | trunc 63 | trimSuffix "-" }}
+  labels:
+    app: {{ template "prometheus-alerts.name" . }}
+    {{- include "prometheus-alerts.labels" . | nindent 4 }}
+    {{- if .Values.defaultRules.alertLabels }}
+    {{- toYaml .Values.defaultRules.alertLabels | nindent 4 }}
+    {{- end }}
+  {{- if .Values.defaultRules.annotations }}
+  annotations:
+    {{- toYaml .Values.defaultRules.annotations | nindent 4 }}
+  {{- end }}
+spec:
+  groups:
+  - name: packets-dropped
+    rules:
+    - alert: FrequentPacketsDroppedToWorkload
+      annotations:
+        description: Number of packets dropped to a workload because no policies matched them
+        summary: Number of packets dropped to a workload
+      expr: |
+        increase(no_policy_drop_counter{type="tw", cluster=~".*", pod_namespace=~".*", exported_pod=~".*"}[1m]) > 0
```

Comment:
how would increase work if we have a constant number of packets dropped over a long period of time?

Reply:
I tried to simulate this case by running curl on an infinite loop with a delay between each request, and it seems like the increase function stays constant also if we have a constant number of dropped packets 

![image](https://user-images.githubusercontent.com/11875804/223766576-04ca6fc0-11b9-4ca7-a227-915c7a49ad0f.png)

Reply:
@crssnd It should behave as **rate** would, in that case as well 
![image](https://user-images.githubusercontent.com/11875804/223769095-176f8504-f9ae-431c-8735-aadaf56327c3.png)